sim_consistency/cli.py

```diff
@@ -76,7 +76,8 @@ def main_model_sim(base):
                                                             unbiased=base.unbiased,
                                                             device=base.device,
                                                             sigma=base.sigma,
-                                                            max_workers=base.max_workers)
+                                                            max_workers=base.max_workers,
+                                                            gromov_parameter=base.gromov_parameter)
```

Comment:
Maybe I am wrong, but I think argparser Gromov arguments are not passed correctly to the construction function.

Reply:
Oh yeah sorry. I first wanted to summarize all Gromov parameter in a single dict, but decided that this is more complex. I will correct it :)